test/NodeTest.cpp

```diff
@@ -113,9 +115,6 @@ TEST(StringNodeTest, StringltOperator) {
 
   ASSERT_FALSE(nodeInt1 < nodeInt2);        // Same id same data
   ASSERT_FALSE(nodeDouble1 < nodeDouble2);  // Same id different data
-  // hash of 1 is greater than 2
-  ASSERT_TRUE(nodeString2 < nodeString1);   // Different id different id
-  ASSERT_FALSE(nodeStruct2 < nodeStruct1);  // completely different
```

Comment:
`std::hash` results should not be compared numerically, as their implementation differs per-platform